help/user-guide/browsing.md

```diff
@@ -26,10 +26,15 @@ The columns to be shown by the table view can be customised, adding/removing col
 ![Table view configuration](/images/user-guide/browsing_table_config.png){:.screenshot}
 
 # Browsing Samples
-You can browse Samples registered in SEEK by Sample Type or by Templates.
+You can browse Samples registered in SEEK by Sample Type or by Experiment Sample Templates.
 * From any page, click on Browse from the top navigation bar
 * Select Samples from the dropdown menu
 
```

Comment:
Suggestion to add the following bullet points: 
* To query by Experiment Sample Templates, click the button on the top right of the screen
* To filter by Sample Type, use the filters on the left.